src/core/thread/mle_types.hpp

```diff
@@ -302,6 +302,16 @@ enum class ChildDuaState : uint8_t
     kUnchanged, ///< The Child registers the same DUA again.
 };
 
+/**
+ * Security Policy
+ *
+ */
+enum
+{
+    kVersionThresholdOffsetVersion = 3, ///< Offset between the Thread Version and the
+                                        ///< Version-thresold valid for Routing.
+};
```

Comment:
It is better to define this enum in class `SecurityPolicy`.

Reply:
Agree. Done.